examples/hazelcast/README.md

```diff
@@ -17,10 +17,10 @@ This is a somewhat long tutorial.  If you want to jump straight to the "do it no
 Source is freely available at:
 * Hazelcast Discovery - https://github.com/pires/hazelcast-kubernetes-bootstrapper
 * Dockerfile - https://github.com/pires/hazelcast-kubernetes
-* Docker Trusted Build - https://registry.hub.docker.com/u/pires/hazelcast-k8s
+* Docker Trusted Build - https://quay.io/repository/pires/hazelcast-kubernetes
```

Comment:
Well, it may be named differently, but it's a trusted build alright. A trusted build means the source `Dockerfile` is made available.